src/spec.md

```diff
@@ -79,6 +79,49 @@ Each `update` which is passed to the callback comes with the following propertie
 - `update.summary`: optional, short text, shown beside icon (see [`sendUpdate()`](#sendupdate))
 
 
+### sendToChat()
+
+```js
+promise = window.webxdc.sendToChat(message);
+```
+
+`sendToChat()` (requires Delta Chat 1.38 or compatible) allows an webxdc app to prepare a message
+that can then be sent to a chat by the user.
+Implementations may ask the user for a destination chat
+and then set up the message as a draft,
+so it is clear that the outgoing message is a result of some user interaction.
+
+- `message`: an object with the following properties:
+  - `message.file.base64`: base64 encoded file data
+  - `message.file.name`: name of the file
+  - `message.text`: message text that should be sent together with the file
```

Comment:
you can also just set the text the wording opposes this reality a bit

Reply:
thanks! i changed that in https://github.com/webxdc/webxdc_docs/pull/49